packages/ruleset/test/utils/root-document.js

```diff
@@ -104,15 +110,32 @@ module.exports = {
       },
       get: {
         operationId: 'listMovies',
+        summary: 'List movies',
+        description:
+          'Retrieve a list of movies using an optional genre qualifier.',
+        parameters: [
```

Comment:
added a param WITHOUT a description to test that the rootDocument validates clean (we don't require a description on a parameter's schema